.github/MIGRATION.md

```diff
@@ -21,10 +21,12 @@
 
 - To reduce the bundle sizes of applications using Stacks.js, we are switching from Buffer (a polyfill to match Node.js APIs) to Uint8Arrays (which Buffers use in the background anyway). [Read more...](#buffer-to-uint8array)
 - To allow message signing on Ledger hardware wallets, we are changing the message signing prefix. [Read more...]
+- Post-conditions for NFTs were renamed to be more clear: `Owns` to `Sends`, `DoesNotOwn` to `DoesNotSend`.
 
 #### Buffer to Uint8Array
 
 To make the switch easier we have introduced a bunch of methods for converting between strings and Uint8Arrays: `hexToBytes`, `bytesToHex`, `utf8ToBytes`, `bytesToUtf8`, `asciiToBytes`, `bytesToAscii`, and `concatBytes`.
+To migrate, update code using Buffers to now use Uint8Arrays.
```

Comment:
I found this wording slightly confusing, paired with the code snippets below. I might suggest:

"To migrate, switch `Buffer` code to instead use `Uint8Array`."

Also, in each code segment below, it might be a little clearer with code comments, like this:

```ts
// old: 
Buffer.from('stacks Ӿ'); // <Buffer 73 74 61 63 6b 73 20 d3 be>

// new:
utf8ToBytes('stacks Ӿ'); // Uint8Array(9) [ 115, 116, 97, 99, 107, 115, 32, 211, 190 ];
```

Reply:
Good call! Better to be verbose. Thx 🙏